dingo/core/utils/pt_to_hdf5.py

```diff
@@ -0,0 +1,77 @@
+import os
+import torch
+import numpy as np
+import h5py
+import json
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Convert the weights of a trained PyTorch model to HDF5 format"
+        " for distribution in the LVK's CVMFS.",
+        epilog="Training history (optimizer_state_dict) is discarded.")
+    parser.add_argument("-i", "--in_file", type=str, required=True,
+            help='Input model ".pt" weights file')
+    parser.add_argument("-o", "--out_file", type=str, required=True,
+            help='Output model ".hdf5" weights file')
+    parser.add_argument("-n", "--version_number", type=int, required=True,
```

Comment:
I suggest to specify "model" version to avoid confusion with Dingo version.
```suggestion
    parser.add_argument("-n", "--model_version_number", type=int, required=True,
```